### PR TITLE
setting up backend-api call

### PR DESCRIPTION
**Pull request by AdarshKumar02291**
Now the data will be served from the api instead of hard coding

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -59,4 +59,6 @@ typings/
 
 .node_modules
 .openode
-# config/
\ No newline at end of file
+# config/
+
```
Comment (DEVAbhinav) [inline .gitignore:63]: Why this change

File: app/package.json [hunk 2]
```diff
@@ -29,7 +29,7 @@
   },
   "scripts": {
     "start": "react-scripts start",
-    "build": "CI=false react-scripts build",
```
Comment (DEVAbhinav) [inline app/package.json:32]: Why this revert?

File: app/src/components/blogs/Blogs.jsx [hunk 3]
```diff
@@ -28,16 +18,13 @@ function Blogs({match}) {
...
-      var url = `/${route}`;
-      var request = {
-        url,
-        method: "get",
-      };
-      const res = await axios(route);
-      const result = await res.data;
-      setData(result);
+      
+      const res = await axios.get(`http://localhost:7071/api/getBlogs?name=${route}`);
```
Comment (DEVAbhinav) [inline app/src/components/blogs/Blogs.jsx:22]: replace the local host with just / routing. No need to provide localhost. It should be picked on default domain by itself.

File: app/src/components/navbar/NavbarComponent.jsx [hunk 4]
```diff
@@ -68,17 +68,17 @@ function NavbarComponent() {
                 <Link to="/blogs/Gaya" className="font-15 text-brown font-regular dropdown-item">
                   Gaya
                 </Link>
-                <Link to="/blogs/Sarnath Darshan" className="font-15 text-brown font-regular dropdown-item">
+                <Link to="/blogs/Sarnath" className="font-15 text-brown font-regular dropdown-item">
```
Comment (DEVAbhinav) [inline app/src/components/navbar/NavbarComponent.jsx:71]: Why these unnecessary change. Lets strictly focus on change we want to do.

File: app/src/components/package/package-info/Package-blog.jsx [hunk 5]
```diff
@@ -20,16 +16,13 @@ function Package({match}) {
...
-      var url = `/${route}`;
-      var request = {
-        url,
-        method: "get",
-      };
-      const res = await axios(route);
-      const result = await res.data;
-      setData(result);
+      
+      const res = await axios.get(`http://localhost:7071/api/getBlogs?name=${route}`);
```
Comment (DEVAbhinav) [inline app/src/components/package/package-info/Package-blog.jsx:20]: same comment of using api. with routes only. No need of local host. etc

File: .gitignore (hunk 1 above)
Comment (AdarshKumar02291) [reply .gitignore:63]: fixed
